api/README.md

```diff
@@ -0,0 +1,37 @@
+[back to hoodie-task-server](../README.md)
+
+# hoodie-task-server/api
+
+> API to manage tasks.
+
+After registering the `@hoodie/task-server` hapi plugin, the `Task` API becomes available at `server.plugins.task.api`, so for example to close a task with a plugin, use `server.plugins.task.api.success(taskDoc)` It can also be required directly.
+
+## Example
+
+```js
+var PouchDB = require('pouchdb')
+var task = require('@hoodie/task-server/api')(PouchDB)
+```
+
+## API
+
+- [Factory](#factory)
+- [Task.success()](#tasksuccess)
+- [Task.error()](#taskerror)
+- [Events](#events)
+
+
+
+```js
+// all methods return promises
+task.success(taskDoc)
```

Comment:
If I recall correctly, we wanted to add an additional property where we can pass in properties that get added to the deleted revision, so we can access them and give more context in the client? So it would be `task.success(taskDoc, additionalProperties)`, `additionalProperties` being optional